webviz_4d/plugins/_surface_viewer_4D/_surface_viewer_4D.py

```diff
@@ -60,6 +60,17 @@ def __init__(
         super().__init__()
         self.shared_settings = app.webviz_settings["shared_settings"]
         self.fmu_directory = self.shared_settings["fmu_directory"]
+
+        try:
+            self.basic_well_layers = self.shared_settings["basic_well_layers"]
```

Comment:
Could this be written as
```python
self.basic_well_layers = self.shared_settings.get("basic_well_layers", None)
``` ?

Reply:
Ja, det vil jo absolutt blir mer kompakt og ryddig, tenkte så vidt tanken selv, men den forsvant plutselig ut av hodet;-). Skal sjekke!